recipes-core/images/cvm-initramfs.bb

```diff
@@ -7,7 +7,7 @@ that can subsequently be picked up by external image generation tools such as wi
 
 CVM_DEPS = "busybox-mdev init-ifupdown initscripts base-files base-passwd netbase busybox-udhcpd"
 
-PACKAGE_INSTALL = "ca-certificates sysvinit busybox-udhcpd date-sync logrotate cronie ${CVM_DEPS} ${VIRTUAL-RUNTIME_base-utils} ${ROOTFS_BOOTSTRAP_INSTALL}"
+PACKAGE_INSTALL = "ca-certificates sysvinit busybox-udhcpd date-sync logrotate cronie ${CVM_DEPS} ${VIRTUAL-RUNTIME_base-utils} ${ROOTFS_BOOTSTRAP_INSTALL} system-api-fake-fifo"
```

Comment:
I wouldn't add it here as this place would serve the base image.
I would recommend either adding it as part of the meta-evm image cvm-initramfs.bbappend file or you can add it as a runtime dependency for the system-api recipe (something like RDEPENDS:{PN} += system-api-fake-fifo).

Reply:
Added system-api-fake-fifo as dependency to system api